microsite/static/js/medium-zoom.js

```diff
@@ -0,0 +1,7 @@
+setTimeout(() => {
+  // Ref: https://github.com/francoischalifour/medium-zoom#options
+  mediumZoom('[data-zoomable]', {
+    margin: 20,
+    background: '#000',
+  });
+}, 100);
```

Comment:
This feels like a hack. :) Could it use some form of onDocumentReady thing instead?

Reply:
Yeeep, that will be better indeed. Done!